synthesizer/src/store/helpers/memory/transaction.rs

```diff
@@ -61,6 +61,17 @@ impl<N: Network> TransactionStorage<N> for TransactionMemory<N> {
         Ok(Self { id_map: MemoryMap::default(), deployment_store, execution_store })
     }
 
+    /// Initializes the transaction storage for testing.
+    #[cfg(feature = "testing")]
+    fn open_testing(path: Option<std::path::PathBuf>, transition_store: TransitionStore<N, Self::TransitionStorage>) -> Result<Self> {
```

Comment:
since this is `TransactionMemory`, shouldn't the `path` be ignored, and all the underlying stores be `MemoryMap::default()`?

Reply:
The intention was to allow it to be resolved by child invocations.
I agree that it should be ignored so by switching to `None`, we could save some unnecessary clones.